Quickstart/Sample/Program.cs

```diff
@@ -1,20 +1,50 @@
-﻿using Microsoft.AspNetCore.Hosting;
+﻿using Auth0.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using SampleMvcApp.Support;
+using System.Net;
 
-namespace SampleMvcApp
+using Microsoft.IdentityModel.Logging;
+
+var builder = WebApplication.CreateBuilder(args);
+
+//Adding these 2 statements prevents .NET 6 from advising you we must declare these in the old ConfigureServices
+//method from previous .NET versions.
+builder.Services.AddAuthorization();
+builder.Services.AddControllers();
```

Comment:
I'm not getting any issues when removing those two. Can you elaborate?

Reply:
Sure, so I've been getting an error similar to #37778. 

It happened on the lines

```csharp
endpoints.MapDefaultControllerRoute();
//AND
app.UseAuthorization();
```
The only way I resolved it was by 
[watching this tutorial from where it starts ](https://www.youtube.com/watch?v=vhNhcuht0J0&t=385s) until 6:56. Sadly I cannot any long replicate the error. 

Tried everything I could think of to replicate it. Rebuilt the binaries, Restarted the computer. I suppose I hadn't rebuilt the project properly when I had that error? So I'm following through with your suggested changes. Moving on to the docs, thanks.

Reply:
Whoops forgot to take ```builder.Services.AddMvc();``` out. Will do that now as that line is no longer resulting in the error which is really really strange. Before that, it fixed an error called.
[No service for type 'Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionaryFactory' has been registered](https://stackoverflow.com/questions/38709538/no-service-for-type-microsoft-aspnetcore-mvc-viewfeatures-itempdatadictionaryfa) the suggested solution worked.

Reply:
Thanks!